R/secure_rmd.R

```diff
@@ -107,6 +107,13 @@ secure_rmd <- function(
 
   yaml_polished <- yaml_header$polished
 
+  if (!is.null(yaml_polished$global_sessions_config$app_name) &&
+      !is.null(global_sessions_config_args$app_name)) {
+    warning(paste0("app_name specified in ",
+                   "global_sessions_config_args and YAML polished header,",
+                   " using global_sessions_config_args$app_name"))
+    yaml_polished$global_sessions_config$app_name <- NULL
```

Comment:
I think we want to do this the other way around, or change which arguments take precedence for all the other arguments.  

Currently all the polished YAML arguments will override the arguments in "global_sessions_config_args", but with this change "app_name" is the only one where the "global_sessions_config_args" value overrides the value in the YAML header.  I am fine switching it up so that "global_sessions_config_args" always trump the YAML args.  Thoughts?

Reply:
Agreed this should be adapted, but I think `global_sessions_config_args` should always override `polished` YAML so that you can change it programmatically.